<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Member Info</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="member_name_label">
               <property name="text">
                <string>Member name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="member_name_line_edit"/>
             </item>
             <item>
              <widget class="QPushButton" name="get_button">
               <property name="text">
                <string>Get</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="age_label">
             <property name="text">
              <string>Age:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="gender_label">
             <property name="text">
              <string>Gender:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="sub_start_date_label">
             <property name="text">
              <string>Subscription start date:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="sub_end_date_label">
             <property name="text">
              <string>Subscription end date:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="message_text_browser"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="manage_database_menu">
    <property name="title">
     <string>Manage Database</string>
    </property>
    <addaction name="register_action"/>
    <addaction name="delete_action"/>
   </widget>
   <addaction name="manage_database_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="register_action">
   <property name="text">
    <string>Register New Member</string>
   </property>
  </action>
  <action name="delete_action">
   <property name="text">
    <string>Delete Existing Member</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
